### ISSUE TITLE
[TIL] 정유태_201015

### ISSUE THREAD

**Issue by yut-a** [codestates/ds-TIL#555]
- **키워드**: `배운 내용에 해당하는 키워드를 2-3개 정도 적습니다 (예: Python, pandas, 시각화, etc.)`

--> `Logistic Regression` `train/validation/test set`
- **배운 것**: `어떤 내용을 배웠는지 간략하게 적습니다` 

1. 예측 모델을 위해 train/validation/test set 나누는 방법
3. odds와 logit의 개념
2. Logistic Regression 모델

- **어려웠던 부분**: `배운 내용 중에서 어렵거나 헷갈렸었던 부분을 작성합니다`

--> 어제에 비해서 상대적으로 어려운 부분은 없었음.

- **더 알고 싶은 부분 / 공부하고 싶은 부분**: `더 알고 싶거나 공부할 필요가 있는 부분에 대해 작성합니다`

--> 이번 주 내용을 잘 정리해야 할 필요가 있음.

- **느낀 점**: `오늘 하루에 대한 감상, 기분 등을 간단하게 작성합니다`

--> 점점 분석에서 활용할 수 있는 모델이 많아져서 좋음. 기존에 알고 있던 내용에 더해지는 것과 덜 이해됐던 부분이 이해되는 것이 좋음.
